runtime/jvmti/jvmti_internal.h

```diff
@@ -900,6 +900,65 @@ jvmtiIterateOverReachableObjects(jvmtiEnv* env,
 
 /* ---------------- jvmtiHelpers.c ---------------- */
 
+/**
+ * Allocate a thread local storage key (which represents a jvmtiEnv) to index into a thread
```

Comment:
`@brief` is missing here and in some of the methods below.
```suggestion
 * @brief Allocate a thread local storage key (which represents a jvmtiEnv) to index into a thread
```